openslides_backend/models/list_of_speakers.py

```diff
@@ -0,0 +1,26 @@
+from ..shared.patterns import Collection
+from . import fields
+from .base import Model
+
+
+class ListOfSpeakers(Model):
+    """
+    Model for list of speakers.
+
+    Reverse fields:
+    - speakers_ids
+    - projection_ids
+    - current_projector_ids
+    """
+
+    collection = Collection("list_of_speakers")
+    verbose_name = "list_of_speakers"
+
+    id = fields.IdField(description="The id of this list of speakers.")
+    meeting_id = fields.RequiredForeignKeyField(
+        description="The id of the meeting of this list of speakers.",
+        to=Collection("meeting"),
+        related_name="list_of_speakers_ids",
+    )
+    closed = fields.BooleanField(description="If the list of speakers is closed.")
+    # TODO content_object_id: */list_of_speakers_id;
```

Comment:
This is just a reverse field.

Reply:
Add this to the reverse fields. Also expand the reverse fields with the model information.